bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestScan.java

```diff
@@ -167,4 +164,55 @@ public void test100ResultsInScanner() throws IOException {
     table.delete(deletes);
     table.close();
   }
+
+  @Test
+  /**
+   * 
```

Comment:
The test looks good. Can you document here a bit, something like "Tests row deletion while iterating through a scan result."